praktikumsplaner-frontend/frontend/src/components/meldezeitraeume/MeldezeitraumList.vue

```diff
@@ -1,50 +1,107 @@
-
-                            <v-icon>mdi-calendar-end</v-icon>
-                            {{
-                                useFormatter().formatDateFromString(
-                                    meldezeitraum.zeitraum.endZeitpunkt
-                                )
-                            }}
-                        </v-list-item-subtitle>
-                    </v-list-item-content>
-                </v-list-item>
-                <v-divider></v-divider>
-            </v-card>
-        </v-list>
-        <slot
-            v-else
-            name="notfoundmessage"
-        ></slot>
-    </v-col>
+    <v-container>
+        <v-col>
+            <slot name="header"></slot>
+            <v-list v-if="props.value.length > 0">
+                <v-card
+                    v-for="meldezeitraum in props.value"
+                    :key="meldezeitraum.id"
+                >
+                    <v-divider></v-divider>
+                    <v-list-item two-line>
+                        <v-list-item-content>
+                            <v-list-item-title class="justify-center">
+                                {{ meldezeitraum.zeitraumName }}
+                                <slot name="card-title-icon"></slot>
+                            </v-list-item-title>
+                            <v-list-item-subtitle>
+                                <v-icon>mdi-calendar-start</v-icon>
+                                {{
+                                    useFormatter().formatDateFromString(
+                                        meldezeitraum.zeitraum.startZeitpunkt
+                                    )
+                                }}
+
+                                <v-icon>mdi-calendar-end</v-icon>
+                                {{
+                                    useFormatter().formatDateFromString(
+                                        meldezeitraum.zeitraum.endZeitpunkt
+                                    )
+                                }}
+                            </v-list-item-subtitle>
+                        </v-list-item-content>
+                        <v-list-item-action>
+                            <v-btn
+                                icon
+                                @click="warnBeforeDelete(meldezeitraum)"
+                            >
+                                <v-icon>mdi-delete</v-icon>
+                            </v-btn>
+                        </v-list-item-action>
+                    </v-list-item>
+                    <v-divider></v-divider>
+                </v-card>
+            </v-list>
+            <slot
+                v-else
+                name="notfoundmessage"
+            ></slot>
+        </v-col>
+        <yes-no-dialog-without-activator
+            dialogtitle="Achtung!"
+            dialogtext="Wenn Sie den Meldezeitraum löschen, werden auch alle zugehörigen Praktikumsplätze gelöscht. Wollen Sie den Meldezeitraum wirklich löschen?"
```

Comment:
In `PraktikumsstelleCardZuweisung` the values are bound to a variable. In my opinion both components should be implemented the same way.